synapse/lib/cell.py

```diff
@@ -1738,48 +1774,37 @@ async def promote(self, graceful=False):
             mesg = 'promote() called on non-mirror'
             raise s_exc.BadConfValu(mesg=mesg)
 
-        ahaname = self.conf.get('aha:name')
-        logger.warning(f'PROMOTION: Performing leadership promotion graceful={graceful} ahaname={ahaname}')
+        logger.warning(f'PROMOTION: Performing leadership promotion graceful={graceful}.')
```

Comment:
I don't understand why the ahaname was removed here?

Reply:
Because aha servers don't have an ahaname